x-pack/docs/en/security/authorization/privileges.asciidoc

```diff
@@ -20,6 +20,10 @@ See <<delete-async-search,delete async search>> API for more informations.
 Privileges to create snapshots for existing repositories. Can also list and view
 details on existing repositories and snapshots.
 
+`enroll`::
+The privilege to call the enroll API to enroll a node in a secured Elasticsearch cluster and to allow a client to
```

Comment:
Let's focus on what we can do with the privileges rather what APIs we can call. For instance

Privilege to join new nodes to a cluster with security features enabled and to configure clients to communicate with a cluster with security features enabled.